pages/glossary.md

```diff
@@ -147,6 +171,14 @@ The threshold of a Safe account is a crucial configuration element that enables
 See also:
 - [Get the threshold](./sdk/protocol-kit/reference.md#getthreshold) and [change the threshold](./sdk/protocol-kit/reference.md#createchangethresholdtx) of a Safe with the Safe{Core} SDK on docs.safe.global
 
+## User Operation
```

Comment:
Should this be UserOperation instead?

Reply:
This is something we still need to clarify. Multiple sources are using it differently
- User Operations
- UserOperations
- user operations

Reply:
Should we ask in #protocol for opinions?

Reply:
Well, these two can co-exist:
- "`UserOperation` objects"
- "user operations"